docs/getting-started/README.md

```diff
@@ -5,7 +5,7 @@ Lens is lightweight and simple to install. You'll be up and running in just a fe
 
 ## System Requirements
 
-Review the [System Requirements](/supporting/requirements/) to check if your computer configuration is supported.
+Review the [System Requirements](../supporting/requirements.md) to check if your computer configuration is supported.
```

Comment:
...but this is in line with other links in this file anyway

Reply:
Ah so it should be this instead?

Reply:
? This is what @rafedramzi proposed, I was merely agreeing it is suitable, even though the original link worked locally.

Reply:
Sorry I didn't explain the problem clearly, I explain the problem in https://github.com/lensapp/lens/pull/2155#issuecomment-781393646